service/resource/legacyv2/resource.go

```diff
@@ -207,7 +213,27 @@ func (s *Resource) NewUpdatePatch(ctx context.Context, obj, currentState, desire
 
 	s.logger.Log("info", fmt.Sprintf("cluster '%s' processed", keyv2.ClusterID(cluster)))
 
-	return &framework.Patch{}, nil
+	patch := framework.NewPatch()
+
+	// cloudformation logic: on creation we need to create the cloudformation resource after legacy so that there are no dependency
+	// problems (see https://github.com/giantswarm/operatorkit/issues/139). Once the transition to cloudformation is done we
+	// will separate the cloudformation and legacy resources.
+	if keyv2.UseCloudFormation(cluster) {
+		create, err := s.newCreateChange(ctx, obj, currentState, desiredState)
+		if err != nil {
+			return nil, microerror.Mask(err)
+		}
+
+		update, err := s.newUpdateChange(ctx, obj, currentState, desiredState)
+		if err != nil {
+			return nil, microerror.Mask(err)
+		}
+
+		patch := framework.NewPatch()
```

Comment:
Redefining patch here causes problems when launching clusters with cloud formation.  No stack is created because the create and update changes aren't passed to the framework.

I found this when testing my master node changes.

Reply:
Very good catch! Feel free to push your fix pls :)

Reply:
Fix is pushed.